static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -43,12 +44,23 @@ const AVATAR_STYLES = {
   color: {
     size: 50,
     title: t('Default Logo'),
-    description: t('The default icon for integrations'),
+    previewText: t('The default icon for integrations'),
+    description: t('Image must be between 255px by 255px and 1024px by 1024px.'),
   },
   simple: {
     size: 20,
     title: t('Default Icon'),
-    description: t('This is an optional icon used for Issue Linking'),
+    previewText: tct(
+      'This is an optional silhouette icon used for [uiDocs:UI Components]',
```

Comment:
Hmmmmmm it both is and isn't optional 🤔 Maybe say:

"This is a silhouette icon required only for UI components" or something like that?

Reply:
@ceorourke Since it is optional for the form, but required for publishing I added some new copy and updated the screenshots.